src/meta-srv/src/procedure/region_failover/failover_start.rs

```diff
@@ -38,6 +39,38 @@ impl RegionFailoverStart {
         }
     }
 
+    async fn failed_region_exist(
+        &self,
+        ctx: &RegionFailoverContext,
+        failed_region: &RegionIdent,
+    ) -> Result<bool> {
+        let table_id = failed_region.table_ident.table_id;
+        let route = ctx
+            .table_metadata_manager
+            .table_route_manager()
+            .get(table_id)
+            .await
+            .context(error::TableMetadataManagerSnafu)?
+            .with_context(|| error::TableRouteNotFoundSnafu {
```

Comment:
Table route not found may not be an error. It could be the table being dropped. I think a warning log is enough.

Reply:
It will be okay if it returns an error if the table was dropped before this procedure acquired the lock.